### ISSUE TITLE
RTXML: Can't do autotick for Clock after display

### ISSUE THREAD

**Issue by r3d9u11** [RealyUniqueName/StablexUI#236]
Hi. I can't do autotick for Clock after display. After launching compiled program it crashes: `Uncaught exception - Invalid call`

Main.hx:
```
package;

import ru.stablex.ui.UIBuilder;
import ru.stablex.ui.RTXml;
import sys.io.File;

class Main extends openfl.display.Sprite{
	static public function main () : Void{
		UIBuilder.init(null, true);
		
		openfl.Lib.current.addChild(
			RTXml.buildFn(File.getContent("/home/eugene/Desktop/Autotick/XmlGui/Gui.xml"))()
		);
	}
}
```
Gui.xml:
```
<?xml version="1.0" encoding="UTF-8"?>
<Clock value="60*60*2" on-display="$this.run();"/>
```

Project: [Click_with_Autotick.zip](https://github.com/RealyUniqueName/StablexUI/files/486759/Click_with_Autotick.zip)

** needed to change path to Gui.xml to your local path

Thanks.

**Comment by RealyUniqueName** [comment]
Fixed in dde63f5

**Comment by r3d9u11** [comment]
works fine now, thank you!
